docs/sphinx/dev_guide/release_process.rst

```diff
@@ -0,0 +1,137 @@
+.. ##
+.. ## Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
+.. ## and RAJA project contributors. See the RAJA/COPYRIGHT file
+.. ## for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+.. ##
+
+.. _release-label:
+
+*******************************************
+RAJA Release Process
+*******************************************
+
+The RAJA release process typically involves the following sequence of steps:
+
+  #. Identify all work (features in development, outstanding PRs, etc.) to be 
+     to be included in the release.
+  #. Merge all PRs containing work to be included in the release into the 
+     develop branch.
+  #. Make a :ref:`rcbranch-label` from the develop branch. Finalize the 
+     release by completing remaining release tasks on that branch.
+  #. When the release candidate branch is ready, make a PR for it to be merged
+     into the **main branch.** When it is approved and all CI checks pass,
+     merge the release candidate branch into the RAJA main branch.
+  #. On GitHub, make a new release with a tag for the release. Following our
+     convention, the tag label should have the format ``vMM.mm.pp``. See 
+     :ref:`semver-label` for a description of the version numbering scheme we 
+     use.  In the GitHub release description, please note key features, 
+     bugfixes, etc. in the release. These should be a high-level summary of the 
+     contents of the ``RELEASE_NOTES.md`` file in the RAJA repo, which may 
+     contain more detailed information. Also, add a note to the 
```

Comment:
This doesn't make sense to me. It sounds confusing (specifically, "if they will not checkout the tag.." part)

Reply:
Fixed.